index.js

```diff
@@ -138,9 +147,8 @@ function run(configPath, options, callback) {
         if(results.length) {
             return results;
         }
-    }).finally(function () {
-        workerFarm.end(workers);
-        process.removeListener('SIGINT', shutdownCallback);
+    }).finally(function() {
+        injectTimeout(finallCallback, 10000);
```

Comment:
Does a smaller number work as well? Is there any real signal that we can hook into to find out the right timeout?

Reply:
I believe getting a result with any number is not guaranteed. we can try smaller numbers though.
For a signal to hook into, I think we should use the new plugin API. The `done` hook would be the best choice, I presume.

Reply:
This plugin is already listening to the done hook. The issue is that webpack-bundle-analyzer is set up wrongly internally, and only executing after the done hook is called.

Reply:
@pago I checked different values for the timeout and it seems it depends on the size of the stats file. I'll try to add the timeout as a flag or config.